### ISSUE TITLE
Sortable Columns

### ISSUE THREAD

**Issue by kraftmatic** [toddpress/songs-list#5]
There is a hard sort on Artist column.  Ideally we should be able to sort any column by clicking on it, but while Streamlit supports this it apparently breaks HTML formatting in the cells causing the link columns to be 'broken'.

**Comment by toddpress** [comment]
oof, that def sounds like my fault lol

**Comment by kraftmatic** [comment]
I don't think so.  It looks like it a deficiency of the library.  Someone did pitch an adequate workaround [here](https://discuss.streamlit.io/t/sort-a-column-of-an-interactive-dataframe-and-send-n-first-sorted-rows-of-an-other-column-to-a-list-back-in-python-ide-to-do-something-with-it/51214/2)

It would just add a dropdown selector to specify what you want to sort by.  If we do this we could add other columns like 'year released' as well.
